README.md

```diff
@@ -56,88 +39,17 @@ This list contains workflows covered by packit tool and links to the documentati
 - [Create a SRPM from the current content in the upstream repository.](https://packit.dev/docs/cli/srpm/)
 - [Sync content of the Fedora dist-git repo into the upstream repository.](https://packit.dev/docs/cli/sync-from-downstream/)
 
-## Configuration
-
-Configuration file for packit is described [here](http://packit.dev/docs/configuration/).
-
-TL;DR
-
-```yaml
-specfile_path: packit.spec
-synced_files:
-  - packit.spec
-upstream_package_name: packitos
-downstream_package_name: packit
-```
-
-## User configuration file
-
-User configuration file for packit is described [here](http://packit.dev/docs/configuration/#user-configuration-file).
-
 ## Requirements
 
 Packit is written in python 3 and is supported only on 3.6 and later.
 
-When packit interacts with dist-git, it uses `fedpkg`, we suggest installing it:
-
-```bash
-sudo dnf install -y fedpkg
-```
-
 ## Installation
 
-On Fedora:
-
-```
-$ dnf install packit
-```
-
-You can also use our [`packit-releases` Copr repository](https://copr.fedorainfracloud.org/coprs/packit/packit-releases/)
-(contains also released versions of [OGR](https://github.com/packit/ogr)):
-
-```
-$ dnf copr enable packit/packit-releases
-$ dnf install packit
-```
-
-Or from PyPI:
-
-```
-$ pip3 install --user packitos
-```
-
-(packit project on PyPI is NOT this packit project)
-
-You can also install packit from `master` branch, if you are brave enough:
+See how to start using packit [here](https://packit.dev/docs/guide/).
 
-You can use our [`packit-master` Copr repository](https://copr.fedorainfracloud.org/coprs/packit/packit-master/)
-(contains `master` version of [OGR](https://github.com/packit/ogr)):
-
-```
-$ dnf copr enable packit/packit-master
-$ dnf install packit
-```
-
-Or
-
-```
-$ pip3 install --user git+https://github.com/packit/packit.git
-```
-
-### Run from git directly:
-
-You don't need need to install packit to try it out. You can run it directly
-from git (if you have all the dependencies installed):
-
-```
-$ python3 -m packit.cli.packit_base --help
-Usage: packit_base.py [OPTIONS] COMMAND [ARGS]...
+## User configuration file
 
-Options:
-  -d, --debug
-  -h, --help         Show this message and exit.
-...
-```
+User configuration file for packit is described [here](http://packit.dev/docs/configuration/#user-configuration-file).
 
 ## Who is interested
```

Comment:
This part is outdated as well -- I think we can remove it because it's not possible to provide current info all the time.

Reply:
> This part is outdated as well -- I think we can remove it because it's not possible to provide current info all the time.

Are you talking about the `Who is interested` part or about `User configuration...` part ?